english/explore-outside/talk-child/talk-more/nice-talk/help-child/abandon-hope/abandon-hope.md

```diff
@@ -0,0 +1,7 @@
+You swim ashore and look back. The yellow lake is quiet. Even the marshmallow boat has melted away and the only proof of there ever having existed a green sad child with a thick Pittsburgh accent is now the image in the back of your mind. Wondering about the child, you look at the car keys in your hand.
+
+Do you:
+
+[Toss the keys into the lake?](/toss-keys/toss-keys.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.